datajunction-clients/python/datajunction/builder.py

```diff
@@ -502,7 +502,7 @@ def create_tag(  # pylint: disable=too-many-arguments
         try:
             self._create_tag(tag=new_tag)
         except DJTagAlreadyExists as exc:
-            if not update_if_exists:
+            if not skip_if_exists:
```

Comment:
How come for tags it's this  flag? Like I could imagine a need to update the tag if it exists, say if the metadata json blob on the tag changes, or the display name changes

Reply:
You are right... and there is a way to do that through tag's `get` and `save` methods. Just these `create_<something>` had a different approach. It was just too much to fix in one PR.

Reply:
It wasn't that bad after all. Changed `create_tag()` to update_if_exists.